### PR TITLE
Add networks api

### PR DESCRIPTION
**Pull request by vetkamak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -191,55 +187,57 @@ import { ProviderRpcClient } from '@eversurf/surfkeeper-provider';
 const rpc = new ProviderRpcClient();
```
Comment (sertony) [inline README.md:187]: Need to remember that further we are to pass the "net" parameter in the constructor... (it looks like this is suitable for API)

File: src/provider.ts [hunk 2]
```diff
@@ -224,37 +230,6 @@ export class ProviderRpcClient {
...
-    /**
-     * Sends transaction with provided params.
-     * Shows an approval window to the user.
-     */
-    public async sendTransaction(
-        args: ProviderApiRequestParams<'sendTransaction'>,
-    ): Promise<ProviderApiResponse<'sendTransaction'>> {
-        await this.ensureInitialized();
-        return this._api.sendTransaction(args);
-    }
```
Comment (sertony) [inline src/provider.ts:256]: What about it?

File: src/provider.ts [hunk 3]
```diff
@@ -63,34 +63,40 @@ export class ProviderNotInitializedException extends Error {
...
+            (api, networkKey) => ({
+                ...api,
+                [networkKey as ProviderNetwork]: new Proxy<RawProviderApiMethods>({} as RawProviderApiMethods, {
+                    get:
+                        <M extends ApiMethod>(_object: RawProviderApiMethods, method: M) =>
+                        (params: RawProviderApiRequestParams<M>) => {
+                            if (this._provider != null && this._provider[networkKey as ProviderNetwork]) {
+                                return this._provider[networkKey as ProviderNetwork].request({ method, params } as {
+                                    method: RequestMethod;
+                                    params: RequestParams<RequestMethod>;
```
Comment (sertony) [inline src/provider.ts:90]: Should we send the "net" in params here? (since we know it)

Comment (vetkamak) [reply src/provider.ts:92]: If I'm not mistaking there is no need to pass network here

**Review by sertony — APPROVED** [review]

**Pull request merged**